### PR TITLE
Updates README.md with new import code examples

### PR DESCRIPTION
**Pull request by joker-777**
* gives an example how to import the library as an es6 module and
  commonjs module
* updated code examples to use `const` instead of `var`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,7 +111,9 @@ audience for the experiments (e.g. mobile users, logged-in etc).
 Experiments automatically start by default. However, a trigger function can be provided, to limit the audience or the
 page(s) where the experiment "kicks-off".
 ```javascript
-var button_color_experiment = new AlephBet.Experiment({
+import {Experiment} from "alephbet"
+
+const button_color_experiment = new Experiment({
   name: 'button color',
   trigger: function() {
```
Comment (gingerlime) [inline README.md:118]: should we also rename the function syntax to use `=>` ?

**Review by gingerlime — APPROVED** [review]
Looks great 👍 thank you for taking the time to update the README.

File: README.md (hunk 1 above)
Comment (joker-777) [reply README.md:118]: True, we could change this. Both would work here though.

**Comment by joker-777** [comment]
Cool, I'm happy that you like it. I changed the functions to arrow functions as well.

**Comment by gingerlime** [comment]
Cool Thank you!

**Pull request merged**